tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -337,13 +337,22 @@ def test_extension_is_not_included(self) -> None:
                 platform=platform,
             )
 
-            with patch(f"{REPO_TREES_CODE}.get_supported_extensions", return_value=["tbd"]):
-                self._process_and_assert_configuration_changes(
-                    repo_trees={REPO1: [file_in_repo]},
-                    frames=[self.frame(frame_filename, True)],
-                    platform=platform,
-                    expected_new_code_mappings=[self.code_mapping("foo/", "src/foo/")],
-                )
+    def test_extension_is_included(self) -> None:
```

Comment:
I separated this test into its two halves preventing the caching from getting on the way.